src/tests/sphdirectpcqueue_ttt.c

```diff
@@ -805,11 +820,18 @@ test_pcq_reset ()
 }
 
 int
-main ()
+main (int argc, char *argv[])
 {
   int rc, rc3;
   double clock, nano, rate;
   sphtimer_t tempt, startt, endt, freqt;
+  int i;
+
+  for (i = 1; i < argc; i++) {
+    cpu_list[i-1] = strtol(argv[i],0,0);
+    if (cpu_list[i-1] > cpu_max) cpu_max = cpu_list[i-1];
```

Comment:
@ThinkOpenly  Should we set cpu_max to sysconf (_SC_NPROCESSORS_ONLN); before this check.